uberon_edit.obo

```diff
@@ -2268,6 +2268,7 @@ xref: VHOG:0000725
 xref: WBbt:0008423
 xref: Wikipedia:Male_reproductive_system_(human)
 xref: XAO:0000155
+is_a: UBERON:0000990 ! reproductive system
```

Comment:
Is this the only change? The rest of the diff seems just spurious (I checked it all, nothing broke, just wanna be sure).

Reply:
sorry. I just saw another inconsistency. so two changes are introduced in this branch.

Reply:
also this shouldn't be necessary as we have the equiv axiom

Reply:
ok. but the other axiom is necessary? I think the first change makes the female and male axioms consistent, right?

Reply:
should I make a new PR?